docs/fundamentals/documentarray/subindex.md

```diff
@@ -217,8 +217,7 @@ Document(embedding=np.random.rand(512)).match(da, on='@c')
 ```
 ````
 
-Such a search will return Documents from the subindex. If you are interested in the top-level Documents associated with
-a match, you can retrieve them using `parent_id`:
+Such a search will return Documents from the subindex. If you are interested in the top-level Documents associated with a match, you can retrieve them using `parent_id`:
```

Comment:
I would remove this sentence, I would say that this is not the recommended usage anymore. Instead people should use the new feature.